### PR TITLE
Created responsive table of contents demo

### PR DESCRIPTION
**Pull request by JianLoong**
This PR attempts to handle issue #15 .

However, because of the nature of design and etc there are potential issues that might arise from this.

What I have done was, introduced **taxonomies** in order for tagging to happen. So each entry in to the glossary will have a tag. For example, "Boilerplate" with the B tag. The table of contents is currently on the left hand side. But the issue here is that it is not in a fixed position. (So, if you scroll if there too much main contents, you might scroll down and over it) 

**Basically how it looks normally**

image

**How it looks on a mobile device** 

image

On the mobile view, there are only the main alphabets.

There are many more improvements that can be potentially made, but I think this is it for now.

TLDR - A demo can be found [here](https://happy-albattani-772d75.netlify.app/) via a netlify demo deployment.

### CODE REVIEW

**Comment by lynmuldrow** [comment]
Hi @JianLoong ! Many apologies for the delay in response to your awesome work- thanks so much for your contribution! Could you please do a pull and update your code to match our glossary at its current state? As it stands, we have some front end improvements and more terms that could serve to test the table of contents further. 

I'd love to incorporate the table of contents, but also want to make sure it's accessible across devices. Looking forward to seeing your solution!
